engine/build.gradle

```diff
@@ -110,6 +110,8 @@ dependencies {
     api group: 'com.miglayout', name: 'miglayout-core', version: '5.0'
     implementation group: 'de.matthiasmann.twl', name: 'PNGDecoder', version: '1111'
 
+    compile group: 'org.terasology', name: 'JNBullet', version: '0.3.2-SNAPSHOT'
```

Comment:
Being that this is one of our internal deps shouldn't it be back in the group where `tera-bullet` was? Also, `compile` is deprecated - should either be `implementation` or `api`